### PR TITLE
Osdev 273 implement filtering by country and status for claims [FE only]

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
FE part of https://github.com/opensupplyhub/open-supply-hub/pull/262

Fix https://opensupplyhub.atlassian.net/browse/OSDEV-273

1. Implement sorting filter for claim status on FE.
2. Implement sortign filter for facility country on FE.
3. Set claim status to 'PENDING' as default filter option on page load.
4. Adjust dynamic URL params in browser search bar to be consistent with get claims API endpoint.
5. Render count of claim response items.

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by roman-stolar — DISMISSED** [review]

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]
Added a request to change some names to be more explicit.

**Comment by mazursasha1990** [comment]
Please, add relevant description to the PR.

File: src/react/src/actions/claimFacilityDashboard.jsx [hunk 1]
```diff
@@ -28,11 +29,14 @@ export const clearFacilityClaims = createAction('CLEAR_FACILITY_CLAIMS');
 export const sortFacilityClaims = createAction('SORT_FACILITY_CLAIMS');
 
 export function fetchFacilityClaims() {
-    return dispatch => {
+    return (dispatch, getState) => {
         dispatch(startFetchFacilityClaims());
 
+        const { filters } = getState();
+        const qs = createQueryStringFromSearchFilters(filters);
```
[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply src/react/src/actions/claimFacilityDashboard.jsx]: Fixed.

File: src/react/src/components/Filters/ClaimStatusFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+function ClaimStatusFilter({
+    claimStatuses,
+    countriesData,
+    claimStatusesOptions,
+    updateClaimStatus,
+    handleClaimStatusUpdate,
+}) {
+    const onChangeClaimStatus = useCallback(
+        s => {
```
[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply src/react/src/components/Filters/ClaimStatusFilter.jsx]: Fixed.

File: src/react/src/components/Filters/ClaimStatusFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}) {
+    return {
+        claimStatuses,
+        claimStatusesOptions,
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+        updateClaimStatus: v => dispatch(updateClaimStatusFilter(v)),
```
[the remark this answers is not in the document]
Comment (VadimKovalenkoSNF) [reply src/react/src/components/Filters/ClaimStatusFilter.jsx]: Fixed.

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Review by roman-stolar — APPROVED** [review]

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Pull request merged**